src/cljs/org/broadinstitute/firecloud_ui/nih_link_warning.cljs

```diff
@@ -0,0 +1,45 @@
+(ns org.broadinstitute.firecloud-ui.nih-link-warning
+  (:require
+   [dmohs.react :as react]
+   [org.broadinstitute.firecloud-ui.common :as common]
+   [org.broadinstitute.firecloud-ui.page.profile :as profile]
+   [org.broadinstitute.firecloud-ui.utils :as utils]
+   ))
+
+
+(defn- get-profile [ajax-args on-done]
+  (let [on-done (if (or (nil? (:canned-response ajax-args)) (:use-canned-response? ajax-args))
+                  on-done
+                  (fn [{:keys [xhr] :as m}]
+                    (utils/cljslog {:status (.-status xhr)
+                                    :statusText (.-statusText xhr)
+                                    :responseText (.-responseText xhr)})
+                    (on-done m)))]
+    (utils/ajax-orch
+     "/profile"
```

Comment:
ah. I was hoping you would use the /nih/status endpoint instead of /profile, so we can centralize the logic for timing into a single place - otherwise, we have that logic duplicated (and therefore possibly out of sync) across UI and orch.